src/components/NameForm.tsx

```diff
@@ -1,9 +1,47 @@
 import React, {ChangeEventHandler, EventHandler, FormEventHandler, MouseEventHandler, SyntheticEvent} from "react";
 import ReactDOM from "react-dom";
+import styled, {keyframes} from "styled-components";
 
 type Props = { callback: (arg: number) => void };
 type State = { value: number };
 
+
+const Button = styled.button`
+display: inline-block;
+background: papayawhip;
+padding: 0.5em;
+margin: 0.5em;
+border: none;
+color: palevioletred;
+  font-weight: bold;`
+
+const Label = styled.label`
+color: palevioletred;
+font-weight: bold;`
+
+const Input = styled.input`
+color: palevioletred;
+font-weight: bold;
+padding: 0.5em;
```

Comment:
we will need to create constants for those values.
But it let's make it later.

Reply:
Ok